src/course.yaml

```diff
@@ -43,6 +43,19 @@ topics:
     questions: intro-to-lending-borrowing.yaml
     status: In Progress
     completionWeek: In Progress
+  - title: How to Lend and Borrow in AAVE
+    key: how-lendings-borrowing-aave
+    details: |
+      .......
```

Comment:
@HARI-124  can you please add couple of paragraphs for this chapter and also the previous one you worked on.